modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala

```diff
@@ -111,7 +111,8 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
     config / resourceGenerators += (config / smithy4sCodegen).map(
       _.filter(_.ext != "scala")
     ),
-    cleanFiles += (config / smithy4sOutputDir).value,
+    config / cleanFiles += (config / smithy4sOutputDir).value,
+    config / cleanFiles += (config / smithy4sResourceDir).value,
```

Comment:
These aren't required to solve the issue, but thought it'd be a good idea.

Added the resource dir and scoped these to be consistent with the rest (everything is scoped by config).

`clean` without qualifiers seems to still work.